editor.planx.uk/src/@planx/components/Checklist/Public.tsx

```diff
@@ -38,6 +39,37 @@ function toggleInArray<T>(value: T, arr: Array<T>): Array<T> {
 }
 
 const ChecklistComponent: React.FC<Props> = (props) => {
+  const autoAnswerableOptions = useStore(
+    (state) => state.autoAnswerableOptions,
+  );
+
+  let idsThatCanBeAutoAnswered: string[] | undefined;
+  if (props.id) idsThatCanBeAutoAnswered = autoAnswerableOptions(props.id);
+
+  if (idsThatCanBeAutoAnswered && idsThatCanBeAutoAnswered.length > 0) {
+    return (
+      <AutoAnsweredChecklist {...props} answerIds={idsThatCanBeAutoAnswered} />
+    );
+  } else {
+    return <VisibleChecklist {...props} />;
+  }
+};
+
+// An auto-answered Checklist won't be seen by the user, but still leaves a breadcrumb
+const AutoAnsweredChecklist: React.FC<Props & { answerIds: string[] }> = (
+  props,
+) => {
+  useEffect(() => {
+    props.handleSubmit?.({
+      answers: props.answerIds,
+      auto: true,
+    });
+  }, []);
+
+  return null;
+};
+
+const VisibleChecklist: React.FC<Props> = (props) => {
```

Comment:
Component types that support automation _or_ being manually put to a user (eg Question & Checklist) now follow this "wrapper" pattern - public component tests, Storybook, etc all still work exactly as before (until/unless we setup mock breadcrumbs state) :+1:

This will let us test automations in multiple ways going forward: 
- Unit tests against the new preview store method `autoAnswerableOptions`
- React component tests that initialise the component & setup mock store state, and check that `handleSubmit` has been called with `"auto": true` or `"auto": false` respectively